specification/communication/data-plane/Common/stable/2022-07-13/common.json

```diff
@@ -84,6 +84,49 @@
         "modelAsString": true
       }
     },
+    "CallingCommunicationIdentifierModel": {
+      "description": "Calling communication identifier model",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        }
+      ],
+      "properties": {
+        "unknownIdentifiers": {
+          "$ref": "#/definitions/UnknownIdentifierModel",
+          "description": "The unknown identifier."
+        }
+      }
+    },
+    "UnknownIdentifierModel": {
+      "description": "Unknown Communication Identifier model",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The unknown id."
+        }
+      }
+    },
+    "CustomContextModel": {
```

Comment:
Let's move this into the incoming_call.jsons

Reply:
where is this located. the incoming call json?